src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/CustomConverterTests/CustomConverterTests.cs

```diff
@@ -180,7 +180,6 @@ void RunTest<TConverterReturn>()
         }
 
         [ActiveIssue("https://github.com/dotnet/runtime/issues/66232", TargetFrameworkMonikers.NetFramework)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/66371", typeof(PlatformDetection), nameof(PlatformDetection.IsMonoInterpreter))] 
```

Comment:
I noticed this issue is already closed

Reply:
The issue was closed by the PR that added the ActiveIssue: https://github.com/dotnet/runtime/pull/66381/files

I would just keep it.